src/core/renderController.ts

```diff
@@ -18,10 +18,10 @@ module Plottable {
    * ```
    */
   export module RenderController {
-    var _componentsNeedingRender = new Utils.Set<Component>();
-    var _componentsNeedingComputeLayout = new Utils.Set<Component>();
-    var _animationRequested = false;
-    var _isCurrentlyFlushing = false;
+    var componentsNeedingRender = new Utils.Set<Component>();
+    var componentsNeedingComputeLayout = new Utils.Set<Component>();
+    var animationRequested = false;
+    var isCurrentlyFlushing = false;
```

Comment:
Agree with @aicioara here -- while it's not necessary to underscore them, it will result in less confusion.